apps/web/components/booking/BookingDescriptionPayment.tsx

```diff
@@ -1,22 +1,35 @@
+import type { TFunction } from "next-i18next";
 import { FormattedNumber, IntlProvider } from "react-intl";
 
 import getPaymentAppData from "@calcom/lib/getPaymentAppData";
 import { FiCreditCard } from "@calcom/ui/components/icon";
 
-const BookingDescriptionPayment = (props: { eventType: Parameters<typeof getPaymentAppData>[0] }) => {
+const BookingDescriptionPayment = (props: {
+  eventType: Parameters<typeof getPaymentAppData>[0];
+  t: TFunction;
+}) => {
   const paymentAppData = getPaymentAppData(props.eventType);
   if (!paymentAppData || paymentAppData.price <= 0) return null;
 
   return (
     <p className="text-bookinglight -ml-2 px-2 text-sm ">
       <FiCreditCard className="ml-[2px] -mt-1 inline-block h-4 w-4 ltr:mr-[10px] rtl:ml-[10px]" />
-      <IntlProvider locale="en">
-        <FormattedNumber
-          value={paymentAppData.price / 100.0}
-          style="currency"
-          currency={paymentAppData.currency?.toUpperCase()}
-        />
-      </IntlProvider>
+      {paymentAppData.paymentOption === "HOLD" ? (
+        <>
+          {props.t("no_show_fee_amount", {
+            amount: paymentAppData.price / 100.0,
+            formatParams: { amount: { currency: paymentAppData.currency } },
+          })}
+        </>
+      ) : (
+        <IntlProvider locale="en">
+          <FormattedNumber
+            value={paymentAppData.price / 100.0}
+            style="currency"
+            currency={paymentAppData.currency?.toUpperCase()}
+          />
+        </IntlProvider>
+      )}
```

Comment:
For "HOLD" I used i18n to format the price instead of the `IntlProvider`. Could we remove `intlProvider` through out the codebase and use i18n instead to reduce our bundle size?

Reply:
This is fine, also hardcoded to "en" is a bug I believe elsewhere, instead i18n.language should be used.